tests/server/blueprints/institutes/test_institute_controllers.py

```diff
@@ -1,5 +1,70 @@
 import copy
-from scout.server.blueprints.institutes.controllers import cases
+from werkzeug.datastructures import MultiDict
+from scout.server.extensions import store
+
+from scout.server.blueprints.institutes.controllers import cases, phenomodel_checkgroups_filter
+
+HPO_TERMS = [
```

Comment:
Any particular reason not to put these in fixtures so they can be reused?

Reply:
Fixed